packages/mgt-components/src/components/mgt-teams-channel-picker/mgt-teams-channel-picker.ts

```diff
@@ -757,6 +757,16 @@ export class MgtTeamsChannelPicker extends MgtTemplatedComponent {
     this.resetFocusState();
   }
 
+  /**
+   * Clears the selectedItem state.
+   *
+   * @memberof MgtTeamsChannelPicker
+   */
+  clearSelectedItem() {
+    this.removeSelectedChannel(undefined);
```

Comment:
I propose that we alter the signature of `removeSelectedChannel` from `removeSelectedChannel(item: ChannelPickerItemState)` to `removeSelectedChannel(item?: ChannelPickerItemState)` that way we don't have to pass null or undefined to clear the state, we just call it with no arguments where we don't know/care what we're removing.

Wes should also update selectChannel to `selectChannel(item?: ChannelPickerItemState)`